app/code/Magento/Inventory/Controller/Adminhtml/Sources/Save.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Controller\Adminhtml\Sources;
+
+/**
+ * Class Edit
+ */
+class Save extends \Magento\Backend\App\Action
+{
+    /**
+     * @var \Magento\Framework\View\Result\PageFactory
+     */
+    protected $resultPageFactory;
```

Comment:
Magento does not encourage inheritance-based API. This property should be private.